packages/stellar-dispatcher/src/index.js

```diff
@@ -0,0 +1,36 @@
+import uuid from 'uuid';
+import _ from 'lodash';
+
+const stoppersMap = {};
+let stellar;
+
+export function configure(_stellar) {
+    stellar = _stellar; // eslint-disable-line better-mutation/no-mutation
+}
+
+export const dispatch = ({ resource, path, method, channel, reactiveHandler, payload, options }) => {
+    if (method === 'unsubscribe') {
+        const stopper = stoppersMap[payload];
+        if (stopper) {
+            stopper();
+        }
+        _.unset(stoppersMap, payload); // eslint-disable-line better-mutation/no-mutating-functions
+        return true;
+    }
+
+    if (method === 'subscribe') {
+        const finalChannel = `op:${_.get(options, 'headers.operationId')}:${channel}`;
+        const response = stellar.request.getReactive(path ? `${resource}:${path}` : resource,
```

Comment:
assign ternary to a const

Reply:
you mean path ? `${resource}:${path}` : resource?